app/connectors/UserAnswersCacheConnector.scala

```diff
@@ -91,7 +92,7 @@ class UserAnswersCacheConnector @Inject()(
       case Some(noEventData) =>
         val headers = eventHeaders(pstr, eventType, Some(noEventData))
         val hc: HeaderCarrier = headerCarrier.withExtraHeaders(headers: _*)
```

Comment:
This line is not required now because you have implicit headerCarrier at definition and you are passing headers to new HttpClientV2 directly.

        val hc: HeaderCarrier = headerCarrier.withExtraHeaders(headers: _*)